prog/dftb+/lib_timedep/rs_linresp.F90

```diff
@@ -3184,4 +3229,54 @@ subroutine getHooT(nOcc, nXov, nXoo, nOrb, homo, nAtom, nMatUp, iaTrans, getIJ,
 
   end subroutine getHooT
 
+  subroutine chargeTestRS(iAtomStart, sTimesGrndEigVecs, grndEigVecs, win, nVir, nOcc, &
```

Comment:
Is there an easy way to have one routine for both chargeTest and chargeTestRS ?

Reply:
That could be done, but since the test routines will go away soon and we anyway merge the LC and non-LC code (at least that is my plan) it is maybe not worth the effort !?

Reply:
That's why I said 'easy'.